src/vorta/application.py

```diff
@@ -325,7 +326,10 @@ def check_failed_response(self, result: Dict[str, Any]):
             if returncode == 1:
                 # warning
                 msg.setIcon(QMessageBox.Icon.Warning)
-                text = self.tr('Borg exited with a warning message. See logs for details.')
+                text = translate(
+                    'VortaApp',
+                    f'Borg exited with a warning message. See <a href="{Path(LOG_DIR).as_uri()}">logs</a> for details.',
+                )
```

Comment:
I noticed a few things, i wasn't aware of before. In my local testing Qt won't recognise the translate statements spanning over multiple lines, for some reason this works when using `self.tr` :shrug:. Additionally, Qt won't be able to translate a string if it changes on runtime. You therefore have to format the string after you translate it.

Reply:
should I try - 
```python
translate('VortaApp','Borg exited with a warning message. See <a href="{0}">logs</a> for details.'.format(Path(LOG_DIR).as_uri()))
```
 - from https://stackoverflow.com/a/18367425

Reply:
Almost. I think the StackOverflow answer is wrong.
```python
translate('VortaApp','Borg exited with a warning message. See <a href="{0}">logs</a> for details.').format(Path(LOG_DIR).as_uri())
 ```